src/jazayeri_lab_to_nwb/watters/watters_convert_session.py

```diff
@@ -19,53 +19,102 @@ def session_to_nwb(data_dir_path: Union[str, Path], output_dir_path: Union[str,
+        assert len(recording_files) == 1, f"Multiple .dat files found in {data_dir_path}"
+        recording_source_data = {
+            f"RecordingVP{probe_num}": dict(
+                file_path=str(recording_files[0]),
+                probe_metadata_file=str(data_dir_path / "data_open_source" / "probes.metadata.json"),
+                probe_key=f"probe{(probe_num+1):02d}",
+                probe_name=f"vprobe{probe_num}",
+                es_key=f"ElectricalSeriesVP{probe_num}",
+            )
+        }
+        raw_source_data.update(recording_source_data)
+        processed_source_data.update(recording_source_data)
+        raw_conversion_options.update({f"RecordingVP{probe_num}": dict(stub_test=stub_test)})
+        processed_conversion_options.update(
+            {f"RecordingVP{probe_num}": dict(stub_test=stub_test, write_electrical_series=False)}
+        )
 
-    source_data = dict()
-    conversion_options = dict()
+        # Add V-Probe Sorting
+        processed_source_data.update(
+            {
+                f"SortingVP{probe_num}": dict(
+                    folder_path=str(data_dir_path / "spike_sorting_raw" / f"v_probe_{probe_num}"),
+                    keep_good_only=False,
+                )
+            }
+        )
+        processed_conversion_options.update({f"SortingVP{probe_num}": dict(stub_test=stub_test, write_as="processing")})
 
     # Add Recording
     recording_files = list(glob.glob(str(data_dir_path / "raw_data" / "spikeglx" / "*" / "*" / "*.ap.bin")))
     assert len(recording_files) > 0, f"No .ap.bin files found in {data_dir_path}"
     assert len(recording_files) == 1, f"Multiple .ap.bin files found in {data_dir_path}"
-    source_data.update(dict(RecordingNP=dict(file_path=str(recording_files[0]))))
-    conversion_options.update(dict(RecordingNP=dict(stub_test=stub_test)))
+    raw_source_data.update(dict(RecordingNP=dict(file_path=str(recording_files[0]))))
+    processed_source_data.update(dict(RecordingNP=dict(file_path=str(recording_files[0]))))
+    raw_conversion_options.update(dict(RecordingNP=dict(stub_test=stub_test)))
+    processed_conversion_options.update(dict(RecordingNP=dict(stub_test=stub_test, write_electrical_series=False)))
 
     # Add LFP
     lfp_files = list(glob.glob(str(data_dir_path / "raw_data" / "spikeglx" / "*" / "*" / "*.lf.bin")))
     assert len(lfp_files) > 0, f"No .lf.bin files found in {data_dir_path}"
     assert len(lfp_files) == 1, f"Multiple .lf.bin files found in {data_dir_path}"
-    source_data.update(dict(LFP=dict(file_path=str(lfp_files[0]), es_key="ElectricalSeriesLF")))
-    conversion_options.update(dict(LFP=dict(write_as="lfp", stub_test=stub_test)))
+    raw_source_data.update(dict(LFP=dict(file_path=str(lfp_files[0]))))
+    processed_source_data.update(dict(LFP=dict(file_path=str(lfp_files[0]))))
+    raw_conversion_options.update(dict(LFP=dict(write_as="lfp", stub_test=stub_test)))
```

Comment:
So, this is actually the one exception where we do not write it as LFP - for SpikeGLX, the LF band is acquired simultaneously  with the AP so it counts as raw acquisition

Most of the time, other systems process this offline

```suggestion
    raw_conversion_options.update(dict(LF=dict(stub_test=stub_test)))
```